src/Nethermind/Nethermind.Merge.Plugin/MergePlugin.cs

```diff
@@ -342,6 +342,12 @@ protected override void Load(ContainerBuilder builder)
                             : NoGCStrategy.Instance,
                         ctx.Resolve<ILogManager>());
                 })
+                .AddSingleton<IHttpClient>((ctx) =>
+                {
+                    IJsonSerializer jsonSerializer = ctx.Resolve<IJsonSerializer>();
+                    ILogManager logManager = ctx.Resolve<ILogManager>();
+                    return new DefaultHttpClient(new HttpClient(), jsonSerializer, logManager, retryDelayMilliseconds: 100);
```

Comment:
Do we need this custom registration?
Can we just go with `.AddSingleton<IHttpClient, DefaultHttpClient>()` and change `retryDelayMilliseconds` to be `100` by default?